browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2442,8 +2450,10 @@ L.Control.JSDialogBuilder = L.Control.extend({
 					builder.map.fire('postMessage', {msgId: 'Clicked_Button', args: {Id: data.id} });
 				else if (isRealUnoCommand && data.dropdown !== true)
 					builder.callback('toolbutton', 'click', button, data.command, builder);
-				else
+				else {
 					builder.callback('toolbox', 'click', parentContainer, data.command, builder);
+                    builder._unoButtonSelection(button, div, parentContainer);
```

Comment:
This patch shouldn't be needed - it means we have to do something wrong.
These buttons come from core, might be some message error.

what does it fix? sorry I don't see any description in the commit message

Reply:
This adds a "selected" state to the active unotoolbutton in the toolbox

Reply:
this should already work, let me check the issue

Reply:
done: https://gerrit.libreoffice.org/c/core/+/181170